tests/src/container-smoke.spec.ts

```diff
@@ -49,6 +50,10 @@ beforeAll(async () => {
   await deleteContainer(page, containerToRun);
 });
 
+beforeEach<RunnerTestContext>(async ctx => {
+  ctx.pdRunner = pdRunner;
```

Comment:
probably pickup `podmanDesktopRunner` because we don't know what shortcut is or `runner` as anyway we have a RunnerTestContext that is dedicated to Podman